vm_insnhelper.c

```diff
@@ -1213,11 +1213,14 @@ vm_setivar(VALUE obj, ID id, VALUE val, const rb_iseq_t *iseq, IVC ic, const str
 	    VALUE *ptr = ROBJECT_IVPTR(obj);
 	    index = !is_attr ? ic->entry->index : vm_cc_attr_index(cc)-1;
 
-	    if (RB_DEBUG_COUNTER_INC_UNLESS(ivar_set_ic_miss_oorange, index < ROBJECT_NUMIV(obj))) {
-		RB_OBJ_WRITE(obj, &ptr[index], val);
-		RB_DEBUG_COUNTER_INC(ivar_set_ic_hit);
-		return val; /* inline cache hit */
-	    }
+	    if (index >= ROBJECT_NUMIV(obj)) {
```

Comment:
trivial: You removed hard tab for L1220 and not for this line. So this looks weirdly indented on GitHub
```suggestion
            if (index >= ROBJECT_NUMIV(obj)) {
```

Reply:
Oops!  Thank you!